packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,155 @@
+        Accordion two content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <span slot="heading">Accordion three</span>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion three content
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+Accordion group, appearance block with a nested accordion
+
+```html
+<nimble-accordion appearance="block">
+    <nimble-accordion-item>
+        <div slot="heading">Accordion one</div>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion one content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion two</div>
+        <nimble-button>Text Button</nimble-button>
+        Accordion two content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion three</div>
+        <nimble-checkbox></nimble-checkbox>
+        <nimble-accordion>
+            <nimble-accordion-item>
+                <div slot="heading">Nested Accordion One</div>
+                Nested Accordion One Content
+            </nimble-accordion-item>
+            <nimble-accordion-item>
+                <div slot="heading">Nested Accordion Two</div>
+                Nested Accordion Two Content
+            </nimble-accordion-item>
+        </nimble-accordion>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion four</div>
+        <nimble-button>Text Button</nimble-button>
+        Accordion four content
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+    -   FAST's attribute `expand-mode` will be omitted from the FAST base class due to issues discussed below.
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. These designs are similar to those of the nimble-button and its `appearance` types, but the accordion does not use the `.control` css class, it uses `.region` and `.heading`. Because of this, differences in border functionality, and the possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion and accordion-item will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. `:host([appearance='outline'])` will have styling for the outline button).
```

Comment:
```suggestion
The design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. These designs are similar to those of the nimble-button and its `appearance` types, but the accordion does not use the `.control` css class, it uses `.region` and `.heading`. Because of this, differences in border functionality, and the possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion and accordion-item will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. `:host([appearance='outline'])` will have styling for the outline button).
```

No reason to call out Figma by name in these specs.